data-scripts/requirements.txt

```diff
@@ -1,5 +1,5 @@
-click==6.7
-petl==1.1.1
-ratelimit==1.4.1
-contentful_management==1.4.0
-tqdm==4.19.2
+click>=6.7,<=8.1.3
+petl==1.1.1,<=1.7.10
```

Comment:
if this has == does the max matter?

Reply:
That's my bad! as long as it's equal to or above the requirements then the scripts work!